action.yml

```diff
@@ -52,9 +74,20 @@ runs:
       uses: GitHubSecurityLab/actions-permissions/monitor@v1
       with:
         config: ${{ inputs.permissions_monitoring_config }}
+
     - id: harden-runner
       if: ${{ inputs.harden_runner == 'true' }}
       name: Harden the runner
       uses: step-security/harden-runner@v2
       with:
         egress-policy: ${{ inputs.harden_runner_egress_policy }}
+
+    - id: check-github-status
+      if: ${{ inputs.check_github_status == 'true' }}
```

Comment:
Are we required to use single quotes around `true` in these conditionals?  If not, we should use double quotes per our usual preference.

Reply:
Yes, [we are required to use single quotes](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#literals).

Reply:
Thanks for the info! 👍